app/services/single_invite_job.rb

```diff
@@ -0,0 +1,25 @@
+class SingleInviteJob
+
+  SingleInviteResult = Struct.new(:error) do
+    def invited_user?
+      error.blank?
+    end
+  end
+
+  def initialize(org, email)
+    @batch_invite = ::BatchInviteJob.new({:resend_invitation => false, :invite_list => {org.id.to_s => email}}, nil)
+    @org_id = org.id
+    @callable_on_error = callable_on_error
```

Comment:
Good catch.  This is actually a vestige of refactoring out the proc callback.  It's not puking because I have attr_reader defined with callable_on_error; when that is removed, it pukes.  Before I was injecting callable_on_error into the initializer, and I think that a local variable takes precedence over an implicitly call to self, as attr_reader so it worked before.  

I'll remove this.